superset-frontend/src/views/CRUD/data/database/types.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+type DatabaseUser = {
+  first_name: string;
+  last_name: string;
+};
+
+export type DatabaseObject = {
```

Comment:
Not sure how we want to hand the `DatabaseObject` between the list and the modal; especially since the modal has some extra fields, I'm not sure if we want to include those as optional props or just create a second type to use in the modal.

Reply:
Do we expect them to be significantly different? If so we can create separate types, or create a base type and extend it for each of the instances. Ideally they wouldn't be too different.